Camera.js

```diff
@@ -83,25 +87,26 @@ export default class Camera extends Component {
       PropTypes.number
     ])
   };
-  
+
```

Comment:
revert these changes to the whitespace before you submit the PR to his repo. I'm only pointing out a few of these, you can find the rest by looking at your PR on SourceTree or other Git tool.